conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
I don't really care, you guys own the repo, I just think the conda recipe and the repo versioning should be in sync. 
Basically, anything released to the public should have an easily identifiable tag. If you want I can make this recipe follow the format of all of our other repos and auto push every successful `master` build to `nightly` conda label. Then you guys only need to worry about tagging for official releases.